Dockerfiles/cluster-agent/datadog-cluster.yaml

```diff
@@ -0,0 +1,50 @@
+
+# The host of the Datadog intake server to send Agent data to
+dd_url: https://app.datadoghq.com
```

Comment:
Is this whole file needed? It's either default values or commented

Reply:
it ain't, I am not using the go generate and I wanted to let the door open for the seds in the entrypoint in case someone wanted to add stuff to it.
It's also to give guidance on what's available.